tests/nightly/jenkins-nightly-run.sh

```diff
@@ -23,6 +23,7 @@ ESX_67_VERSION="ob-8169922"
 VC_67_VERSION="ob-8217866"
 
 DEFAULT_LOG_UPLOAD_DEST="vic-ci-logs"
+DEFAULT_VCH_BRANCH=""
```

Comment:
We could straight up fail if `VCH_BRANCH` isnt' set. This requires us to upload to two locations:
```
${VCH_BRANCH:?Branch must be set}
```

I suspect we want to unify the upload/download path construction in a utility function at some point, but I've no idea if drone can use the output of commands or shell functions in the `target` field. Defer this enhancement from this PR for now.